crates/wasi-http/src/struct.rs

```diff
@@ -2,6 +2,12 @@ use crate::types::{Method, Scheme};
 use bytes::Bytes;
 use std::collections::HashMap;
 
+#[derive(Clone)]
+pub struct Stream {
```

Comment:
Looking at the usages of this struct, it seems that a `From<Bytes>` implementation could really clear this up

Reply:
added.